lib/interpreters/multi_trips.rb

```diff
@@ -21,14 +21,14 @@
 module Interpreters
   class MultiTrips
     def expand(vrp)
-      vrp.vehicles = vrp.vehicles.collect{ |vehicle|
+      vrp.vehicles = vrp.vehicles.flat_map{ |vehicle|
         if vehicle.trips > 1
           new_ids = Array.new(vehicle.trips) { |index| vehicle.id + '_trip_' + index.to_s }
-          new_vehicles = new_ids.collect{ |id|
+          new_vehicles = new_ids.collect.with_index{ |id, index|
             new_vehicle = Marshal.load(Marshal.dump(vehicle))
-            new_vehicle.original_id = new_vehicle.id
             new_vehicle.id = id
             new_vehicle.trips = 1
+            new_vehicle.force_start = false unless index.zero?
```

Comment:
Do we want this? What if I have 5 trips but I want them all to be as early as possible, can't I do that with setting `force_start = true`? I don't know the answer by the way, just brain storming

Reply:
You can do it but then you would have to provide 5 vehicles with independent timewindows and link those vehicles with `vehicle_trips` relation. 

If you provide `trips` field to one vehicle then every trip will have same timewindow and try to start at the same time if `force_start` is activated for all of them.

Reply:
Yes the timing will be the same but this doesn't prevent us, no? The new force_start implementation should be flexible enough to accommodate multiple vehicles with the same time window start and force_start=true with order constraints (trips) between them. Did we try?

In my opinion, the `trips` shortcut functionality is there to help the user to copy/paste their vehicles easily and in this copy/paste logic if do such modifications, then a user who really have 5 identical vehicles for trips need to duplicate their vehicles manually instead of being able to use `trips`.

Reply:
Do you mean new implementation of force start will be able to adjust one vehicle start according to previous trip end? If this is the case then I agree, we can use force_start on every vehicle!

Reply:
No it doesn't but it doesn't need to.

All `force_start` does (this is from memory, you should check it) is to `minimize (actual_vehicle_start - vehicle_timwindow_start)`. So  second vehicle having the same time window start as the first vehicle doesn't change anything, they both get minimized by the solver and both vehicles leave the depot as early as possible (again need to test it with a realistic instance but I see no reason for it not to work). 

Actually thanks to this we might have less gap in between vehicles but not sure :thinking: